chainerx_cc/chainerx/routines/activation.cc

```diff
@@ -0,0 +1,41 @@
+#include <vector>
+
+#include "chainerx/array.h"
+#include "chainerx/backend.h"
+#include "chainerx/backprop_mode.h"
+#include "chainerx/backward_builder.h"
+#include "chainerx/backward_context.h"
+#include "chainerx/constant.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/graph.h"
+#include "chainerx/kernels/math.h"
+#include "chainerx/macro.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/math.cc"
+#include "chainerx/routines/math.h"
+#include "chainerx/routines/manipulation.h"
+#include "chainerx/routines/type_util.h"
+#include "chainerx/scalar.h"
+#include "chainerx/shape.h"
+#include "chainerx/strides.h"
+
+namespace chainerx {
+Array ClippedRelu(const Array& x, Scalar z) {
+    return Minimum(Maximum(0, x), z);
+}
+
+Array Crelu(const Array& x, nonstd::optional<int8_t> axis) {
+    std::vector<Array> c{x, Negative(x)};
+    Array concat = Concatenate(c, axis);
+    return Relu(concat);
+}
+
+Array Elu(const Array& x, Scalar alpha) {
+    Array zero = ZerosLike(x, x.device());
+    Array out = ZerosLike(x, x.device());
+    IfGreaterElseImpl(x, zero, x, alpha * (Exp(x) - 1), out);
```

Comment:
We have a `chainerx::Expm1` now. Could you use it here?

Reply:
Does this line compile? `IfGreaterElseImpl` is "hidden" inside `math.cc` and the signature won't match. I think you actually need to implement a version of `IfGreaterElse` that matches our needs here.